src/main/kotlin/net/atos/zac/smartdocuments/SmartDocumentsService.kt

```diff
@@ -115,3 +116,21 @@ class SmartDocumentsService @Inject constructor(
         ).resultList.toSet().toREST()
     }
 }
+
+/**
+ * Validates that all entries in a REST request are part of pre-defined superset of groups and templates.
```

Comment:
Maybe I am overlooking things but where is the REST request you mention here in this function? From reading this I would expect a function param for the REST request?

Reply:
Changed the JavaDoc to clarify that we validate RESTSmartDocumentsTemplateGroup against another RESTSmartDocumentsTemplateGroup